functests/11-test-os-imaes-namespace_rbac.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+SCRIPTPATH=$(dirname $(readlink -f $0))
+source ${SCRIPTPATH}/testlib.sh
+
+RET=1
+TEST_NS="kubevirt-os-images"
+EDIT_ROLE="os-images.kubevirt.io:edit"
+VIEW_ROLE="os-images.kubevirt.io:view"
+TEST_SA="testsa"
+
+find_namespace() {
+  # Find if the new namespace is created
+
+  local wait_secs=${2:-2}
+  local max_tries=${3:-15}
+
+  for idx in $(seq 1 ${max_tries}); do
+    ((${V} >= 1)) && echo "Attempt ${idx}/${max_tries} to find namespace ${TEST_NS}"
+    oc get ns | grep ${TEST_NS}
+    if [ $? == 0 ]; then
+      ((${V} >= 1)) && echo "Found Namespace in Attempt ${idx}"
+      return 0
+    fi
+    sleep ${wait_secs}s
+  done
+  return 1
+}
+
+find_editclusterrole() {
+  # Find if the Edit Clusterrole is created
+
+  local wait_secs=${2:-2}
+  local max_tries=${3:-15}
+
+  for idx in $(seq 1 ${max_tries}); do
+    ((${V} >= 1)) && echo "Attempt ${idx}/${max_tries} to find edit clusterrole ${EDIT_ROLE}"
+    oc get clusterrole | grep ${EDIT_ROLE}
```

Comment:
Similar to Andrej's comment above, you can `get` the cluster-role by name instead of using `grep`